standalone/src/models/discount-group/fields-config.ts

```diff
@@ -0,0 +1,38 @@
+import { fake, sequence, type TModelFieldsConfig } from '@/core';
+import { createRelatedDates } from '@/utils';
+import { ClientLogging, LocalizedString } from '../commons';
+import type { TDiscountGroupGraphql, TDiscountGroupRest } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const commonFieldsConfig = {
+  id: fake((f) => f.string.uuid()),
+  createdAt: fake(getOlderDate),
+  createdBy: fake(() => ClientLogging.random()),
+  lastModifiedAt: fake(getNewerDate),
+  lastModifiedBy: fake(() => ClientLogging.random()),
+  key: fake((f) => f.string.alphanumeric({ length: { min: 2, max: 256 } })),
+  version: sequence(),
+  sortOrder: fake((f) =>
+    String(f.number.float({ min: 0.00000001, max: 0.99999999 }))
+  ),
+};
+
+export const restFieldsConfig: TModelFieldsConfig<TDiscountGroupRest> = {
+  fields: {
+    ...commonFieldsConfig,
+    name: fake(() => LocalizedString.random()),
+    description: fake(() => LocalizedString.random()),
```

Comment:
We are following the convention of only populating the required fields by default.

I know, we must document this.